app/businessrates/authorisation/config/microserviceGlobal.scala

```diff
@@ -33,10 +32,9 @@ class GuiceModule(
     bind(classOf[OrganisationAccounts]).to(classOf[BackendConnector])
     bind(classOf[PersonAccounts]).to(classOf[BackendConnector])
     bind(classOf[WithIds]).to(classOf[VoaIds])
-    bind(classOf[DB]).toProvider(classOf[MongoProvider]).asEagerSingleton()
   }
 }
 
-class MongoProvider @Inject()(reactiveMongoComponent: ReactiveMongoComponent) extends Provider[DB] {
-  override def get(): DB = reactiveMongoComponent.mongoConnector.db()
+class MongoProvider @Inject()(mongoComponent: MongoComponent) extends Provider[MongoComponent] {
```

Comment:
I don't think this is necessary.  I believe just registering the mongo module will automatically have a provider for mongo component. I've not seen this in other services and it works fine without this